src/Module.php

```diff
@@ -207,14 +208,23 @@ public function seePsalmVersionIs(string $operator, string $version): bool
             $currentVersion = '9999999-dev';
         }
 
-        $version = $parser->normalize($version);
+        $result = $parser->parseConstraints($versionConstraint)
```

Comment:
I think `Composer\Semver\Semver::satisfies()` is a better option here, given how the method has changed.

Reply:
Yep, you're right. I changed it.